server-testing-mini/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
If the files are already tracked `.gitignore` file will not override this. You will need to use `git rm -r --cached node_modules`